README.md

```diff
@@ -6,52 +6,75 @@
 - Package: `stactools.usda_cdl`
 - PyPI: https://pypi.org/project/stactools-usda-cdl/
 - Owner: @pholleway
-- Dataset homepage: http://example.com
+- Dataset homepage: https://www.nass.usda.gov/Research_and_Science/Cropland/Release/index.php
 - STAC extensions used:
+  - [item-assets](https://github.com/stac-extensions/item-assets)
```

Comment:
Though we don't include the classification information yet, we do include the schema in `stac_extensions`, so we should list it here as well.

Reply:
added